molecule/default/tests/test_create_and_assign_floating_ip.py

```diff
@@ -56,21 +39,35 @@ def test_create_floating_ip(host):
 def test_assign_floating_ip_to_instance(host):
     """Assign floating IP to an instance/server"""
 
-    create_instance_from_image(host)
+    # Creating an instance from image
+    data = {
+        "instance_name": instance_name,
+        "from_source": 'image',
+        "source_name": image_name,
+        "flavor": flavor,
+        "network_name": private_net,
+    }
+
+    helpers.create_instance(data, host)
+    sleep(30)  # TODO: will find out a better way to avoid implicit sleep.
+
+    # Verify the new instance is ACTIVE and Running.
+    assert (helpers.get_expected_value('server', instance_name, 'status', 'ACTIVE', host, 20))
+    assert (helpers.get_expected_value('server', instance_name, 'OS-EXT-STS:power_state', 'Running', host, 20))
 
-    assert (floating_ip_name is not None)  # floating_ip_name is same as the floating IP address
+    assert floating_ip_name  # floating_ip_name is same as the floating IP address
```

Comment:
Your comment here indicates that this variable should be renamed to `floating_ip` to prevent confusion. I understand that in the implementation of the helper it is mapping to the 'name' field. However, in the calling code it is confusing.

Reply:
it makes sense, thanks @johnduarte 
updated!